helm/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "helm",
+  "maintainer": "jack@cresta.ai",
+  "manifest_version": "1.0.0",
+  "name": "helm",
+  "metric_prefix": "helm.",
+  "metric_to_check": "helm.total_releases",
+  "creates_events": false,
+  "short_description": "Reports metrics on helm chart status",
+  "guid": "912f4a95-7f68-48c7-8a7e-655c9e6eee0f",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-helm Integration",
+  "categories": [
+    ""
```

```suggestion
    "orchestration",
    "containers",
    "configuration & deployment",
```
Some suggested categories